app.js

```diff
@@ -23,18 +23,17 @@ var google_analytics_id = process.env.GA_ID || null;
 function addGa(html) {
     if (google_analytics_id) {
         var ga = [
-            "<script type=\"text/javascript\">",
-            "var _gaq = []; // overwrite the existing one, if any",
-            "_gaq.push(['_setAccount', '" + google_analytics_id + "']);",
-            "_gaq.push(['_trackPageview']);",
-            "(function() {",
-            "  var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;",
-            "  ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';",
-            "  var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);",
-            "})();",
-            "</script>"
-            ].join("\n");
-        html = html.replace("</body>", ga + "\n\n</body>");
+			"<!-- Google tag (gtag.js) -->",
+			"<script async src=\"https://www.googletagmanager.com/gtag/js?id=" + google_analytics_id + "\"></script>",
+			"<script>",
+			"  window.dataLayer = window.dataLayer || [];",
+			"  function gtag(){dataLayer.push(arguments);}",
+			"  gtag('js', new Date());",
+			"\n",
+			"  gtag('config', " + google_analytics_id + ");",
+			"</script>"
+			].join("\n");
+        html = html.replace("<head>", "<head>\n\n" + ga);
```

Comment:
Oh, yeah, this has been broken for a while hasn't it. I probably would have just removed it, but this is fine too.